openedx/features/partners/models.py

```diff
@@ -18,6 +19,9 @@ def __unicode__(self):
     class Meta:
         verbose_name = "Partner"
         verbose_name_plural = "Partners"
+        permissions = (
+            ("can_access_g2a_performance", "Can access g2a performance"),
```

Comment:
Can we rename this to something more suitable since it is going in `partners` model, and is not specifically for `g2a`.

Reply:
It's specific for g2a.

Reply:
Since the model represents not only g2a but all the white labelled partners, I highly recommend the use of a more generic permission here that might be reused for other partners’ admin stuff in the future, instead of naming it g2a specific and making it single use

Reply:
@mabdullahz actually if we have to specific permissions, otherwise give2asia admin would able to access other partner's performance page as well.

Reply:
BTW, we have changed to code to create permission on post_save of Partner you can recheck.